Automattic/jetpack
# Search: Separate record meter styles by component

This PR separates styles into their own component stylesheets

#### Changes proposed in this Pull Request:
* creates `notice-box.scss`
* creates `record-count.scss`
* moves relevant styles out of `style.scss` and into above separate stylesheets

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Go to /wp-admin/admin.php?page=jetpack-search&features=record-meter on your test site with Search enabled.
* Check there are no weird styling regressions or things that look unusual happening with the record meter, record count, and notice box. 

![image](https://user-images.githubusercontent.com/30754158/173487869-ff4a7738-c648-4ed0-a9dc-5c269256152c.png)
![image](https://user-images.githubusercontent.com/30754158/173487936-7998b68f-f8e1-42c7-ada8-3ed2da4ff3b4.png)
![image](https://user-images.githubusercontent.com/30754158/173487944-6208ebc9-5697-4ebd-837a-98c87a0dd9ad.png)
